metricbeat/module/elasticsearch/_meta/README.md

```diff
@@ -0,0 +1,62 @@
+# Elasticsearch Module
+
+- [About the elasticsearch metricbeat module](https://www.elastic.co/guide/en/beats/metricbeat/current/metricbeat-module-elasticsearch.html)
+- [List of exported fields](https://www.elastic.co/guide/en/beats/metricbeat/current/exported-fields-elasticsearch.html) stored by this module generated from fields.yml files
+
+## Metricsets
+
+Metricbeat will call the following Elasticsearch API endpoints corresponding to each metricset.  The module stores a subset of the response in fields which can be found [here](https://www.elastic.co/guide/en/beats/metricbeat/current/exported-fields-elasticsearch.html).  The Metricbeat exported fields generally follw the same paths as the elasticsearch API response, but may differ somewhat according to the [schema in each metricset](https://github.com/elastic/beats/blob/main/metricbeat/module/elasticsearch/node/data.go#L36).  They are [namespaced with `{module}.{metricset}`](https://github.com/elastic/beats/blob/main/metricbeat/module/elasticsearch/cluster_stats/data.go#L39).
+### ccr
+- `/_ccr/stats`
+- [api reference](https://www.elastic.co/guide/en/elasticsearch/reference/current/ccr-get-stats.html)
+- [mb exported fields](https://www.elastic.co/guide/en/beats/metricbeat/current/exported-fields-elasticsearch.html#_ccr)
```

Comment:
For these links I wonder if we should reference the git assets directly to avoid any possible discrepancies between this readme (in main) and the docs (from the most recent release).

I'd guess @elastic/obs-docs would know if there's any nice ways to cross reference these files with published documentation.

If not we can merge this as-is for sure. Better to have something documented than have nothing while we wait for something perfectly linked up.

Reply:
I am thinking the more common use case for these docs is for SDH purposes than development.  The yaml file is difficult to read.  I did add a comment in the docs that the mb exported fields are created from the yaml files, so a developer can look in there if they want:

> [List of exported fields](https://www.elastic.co/guide/en/beats/metricbeat/current/exported-fields-elasticsearch.html) stored by this module generated from fields.yml files

Reply:
Maybe for now we can just put {stack version} in place of "current" in those URLs to help tip developers off that you should be intentional there? Would also love to hear docs team's thoughts here. I'm also fine with merging this as-is but curious what the best pattern is for this kind of thing.